art/0001-hybris-Silence-some-build-errors-to-let-the-systemim.patch

```diff
@@ -1,4 +1,4 @@
-From 1a27bd48255a96853e49d25c82a0fe47dd477f83 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Please first cleanup the patches so it makes finding/reverting the change easier:) Cf. https://github.com/mer-hybris/hybris-patches/pull/11/commits

Reply:
@sledges done